### ISSUE TITLE
Release-please aborted due to untagged, merged release PRs

### ISSUE THREAD

**Issue by mountaindude** [google-github-actions/release-please-action#443]
### TL;DR

"‼ There are untagged, merged release PRs outstanding - aborting" is the last line of release-please output in GitHub Actions log. Happens in RPA v3 but not in v2.

I kind of understand what an untagged release PR is... but I can't find any such PRs.
And how can I fix the issue? And avoid it happening again?

It has happened for several repos when moving from v2 to v3 of Release-Please-Action, leaving those repos in a less than desired state when it comes to CI workflows.

### Expected behavior

1. I expect a release PR to be created, based on the commit messages I've used since last release was done.
2. I expect the log output from RPA to reflect the config I use in the Action YAML file.
3. I expect the PR title pattern to be taken from the config in the Action YAML rather than from elsewhere

### Observed behavior

1. RPA runs but eventually aborts with "‼ There are untagged, merged release PRs outstanding - aborting"
2. The logs indicate that some other pull request title pattern is used. Note: I *do* have a RPA manifest file in the repo (from previous tests with using manifest files), but with "release-type: simple" that manifest file should not be used by RPA. 
3. The log complains TWICE about pullRequesttitlePattern not being correct. But I am not even specifying this in the Action YAML config.

### Action YAML

```yaml
Note: steps.extract_sp.outputs.servicepack is set in previous step. 
If the branch name triggering the action to run is "dev/sp3", servicepack will be "sp3".
-------------------------

      - name: Run Release-Please
        uses: google-github-actions/release-please-action@v3
        id: release
        with:
          release-type: simple
          package-name: TEST_${{ steps.extract_sp.outputs.servicepack }}
          default-branch: dev/${{ steps.extract_sp.outputs.servicepack }}
          monorepo-tags: false
          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"docs","section":"Documentation","hidden":false},{"type":"refactor","section":"Refactored","hidden":false},{"type":"doc","section":"Documentation","hidden":false},{"type":"chore","section":"Miscellaneous","hidden":false},{"type":"ci","section":"CI/CD","hidden":false}]'
          fork: false
          clean: true
```

### Log output

```text
Run google-github-actions/release-please-action@v3
  with:
    release-type: simple
    package-name: TEST_sp5
    default-branch: dev/sp5
    monorepo-tags: false
    changelog-types: [{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"docs","section":"Documentation","hidden":false},{"type":"refactor","section":"Refactored","hidden":false},{"type":"doc","section":"Documentation","hidden":false},{"type":"chore","section":"Miscellaneous","hidden":false},{"type":"ci","section":"CI/CD","hidden":false}]
    fork: false
    clean: true
    token: ***
    bump-minor-pre-major: false
    bump-patch-for-minor-pre-major: false
    draft: false
    draft-pull-request: false
  env:
    GITHUB_REF: refs/heads/dev/sp5
    GITHUB_EVENT_NAME: push
√ Looking for latest release on branch: dev/sp5 with prefix: 
‼ Could not find releases.
√ Building releases
√ Building strategies by path
√ Looking at files touched by path
√ Building release for path: .
‼ pullRequestTitlePattern miss the part of '${scope}'
‼ pullRequestTitlePattern miss the part of '${component}'
‼ pullRequestTitlePattern miss the part of '${version}'
× Bad pull request title: 'Pending UAT release : 🔖 sp5 release TEST_sp5 3.0.1'
√ Looking at files touched by path
‼ pullRequestTitlePattern miss the part of '${scope}'
√ Building release for path: .
‼ pullRequestTitlePattern miss the part of '${component}'
‼ pullRequestTitlePattern miss the part of '${version}'
× Bad pull request title: 'Pending UAT release (dev/sp5): 🔖 sp5 release TEST_sp5 2.2.0'

√ Looking for latest release on branch: dev/sp5 with prefix: 
‼ Could not find releases.
√ Building pull requests
√ Building strategies by path
√ Collecting release commit SHAs
‼ Could not find releases.
‼ Expected 1 releases, only found 0
‼ Missing 1 paths: .
‼ No version for path .
‼ Expected 1 releases, only found 0
√ Collecting commits since all latest releases
√ Splitting 500 commits by path
√ Building candidate release pull request for path: .
‼ No latest release pull request found.
√ Considering: 49 commits
‼ There are untagged, merged release PRs outstanding - aborting
```

### Additional information

I've battled this for some time now, trying to use manifests and finally now also deleted all previous tags and releases. 
Which makes the scan through the repo to take quite some time, which is reflected in the log output above.

**Comment by bcoe** [comment]
@mountaindude sorry for this nuisance, I've bumped into this myself recently I believe.

Could I bother you to open a ticket on the main [release-please](https://github.com/googleapis/release-please) repository?

**Comment by tmatsuo** [comment]
The upstream issue is fixed. Closing?

**Comment by ddennis** [comment]
I am also getting this issue in my github action, but am unsure how to fix it? 
I have tried the same steps as mentioned above. 

I am alittle unsure if it could be caused by my **.release-please-manifest** or the **release-please-config**

They look like this:
````
.release-please-manifest:
{
  "backend": "1.1.0",
  "frontend": "1.1.0"
}
````


````
release-please-config:

{
  "packages": {
    "backend": {
      "changelog-path": "CHANGELOG.md",
      "bump-minor-pre-major": false,
      "bump-patch-for-minor-pre-major": false,
      "draft": false,
      "release-type": "node",
      "prerelease": false,
      "package-name": "package.json"
    },
    "frontend": {
      "changelog-path": "CHANGELOG.md",
      "bump-minor-pre-major": false,
      "bump-patch-for-minor-pre-major": false,
      "draft": false,
      "release-type": "node",
      "prerelease": false,
      "package-name": "package.json"
    }
  }
}

````
Any help would be much appreciated.
